testing/btest/language/set.zeek

```diff
@@ -150,7 +150,7 @@ function basic_functionality()
 	local a_and_b = a & b;
 
 	test_case( "union", a_or_b == a_plus_b );
-	test_case( "intersection", a_and_b == a_plus_b );
+	test_case( "intersection", a_and_b == a_also_b );
```

Comment:
Given that the bug manifests only depending on the order of arguments, it would be good to add a case that looks for `b & a` in addition to `a & b`

Reply:
added!